containers/unit_tests/CMakeLists.txt

```diff
@@ -13,20 +13,7 @@ foreach(Tag Threads;Serial;OpenMP;HPX;Cuda;HIP;SYCL)
     set(dir ${CMAKE_CURRENT_BINARY_DIR}/${dir})
     file(MAKE_DIRECTORY ${dir})
     foreach(Name
-        Bitset
-        DualView
-        DynamicView
-        DynViewAPI_generic
-        DynViewAPI_rank12345
-        DynViewAPI_rank67
-        ErrorReporter
-        OffsetView
-        ScatterView
-        StaticCrsGraph
         WithoutInitializing
-        UnorderedMap
-        Vector
-        ViewCtorPropEmbeddedDim
```

Comment:
I suppose you didn't mean to commit this

Reply:
Fixed.